### ISSUE TITLE
Cannot require('render-vendor')

### ISSUE THREAD

**Issue by evil-shrike** [isleofcode/render-vendor#2]
1. installed `npm i render-vendor` into an empty npm-project (node v6.8, Windows 10):
```
  "dependencies": {
    "render-vendor": "^0.1.0"
  }
```
2. created build.js:
```
var Renderer = require('render-vendor');
console.log("imported");
Renderer.load('../../_dist/devel/docs/index.html')
  .then((page) => page.render('doc.pdf'))
  .then(() => Renderer.shutdown());
```
3. run 'node build.js'
4. getting "Cannot find module 'babel-runtime/regenerator'":

```
>node build.js
module.js:471
    throw err;
    ^

Error: Cannot find module 'babel-runtime/regenerator'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (C:\Work\R-n-D\xfw3_webclient-docsite\tools\pdf-export\node_modules\render-vendor\renderer.js:8:20)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
```

**Comment by anulman** [comment]
Thanks @evil-shrike. Can you try upgrading to 0.1.2 and let me know if this persists?

This looks like an 0.1.0 issue that was resolved in 0.1.1. Seems 0.1.1 was swallowed somewhere (npm publish says it exists, but I don't recall unpublishing!), so just published 0.1.2.

**Comment by anulman** [comment]
@evil-shrike Closing this issue due to age. Also, just published 0.2.0, which has support for Headless Chrome :)
